runelite-client/src/main/java/net/runelite/client/ui/overlay/worldmap/WorldMapPoint.java

```diff
@@ -50,6 +50,8 @@
 
 	private boolean currentlyEdgeSnapped;
 
+	private WorldMapSnapEdge snappedEdge;
```

Comment:
Why world point needs this information

Reply:
This is stored so that the renderer knows how to move the point later. It's also so the Point can make changes to itself based on its current snapped edge, if it wants to.

Reply:
You can just set some transformation to point, and renderer is not even using it. This is unnecessary.

Reply:
[It's used to fire the onSnapEdgeChanged function](https://github.com/runelite/runelite/pull/3108/commits/711cd5756a1ee3ce83f9e7afe41d221c3565d03a#diff-70d583be06114cef511b6440db1bc78bR152), and it's [called later if the item is snapped](https://github.com/runelite/runelite/pull/3108/commits/b9d3db4069bdcd8dce1dca59b50d7d2787991117#diff-70d583be06114cef511b6440db1bc78bR181) (and though that can be done without storing the variable, if it's used for onSnapEdgeChanged it might as well be used here)